### PR TITLE
Generate Nodejs boilerplate code

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: handlers/boilerplatecode.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Handler, Context } from "aws-lambda";
+import { download } from "../services/compileGRPCStubs";
+
+export const create: Handler = async (
```
Comment (Vivek205) [inline handlers/boilerplatecode.ts:4]: wrap the return type with Promise e.g. `Promise<Handler>` for async methods

File: services/compileGRPCStubs/template/downloadStubFiles.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+});
+
+var params = {
+  localDir: "./grpc_stubs/",
+  s3Params: {
+    Bucket: process.env.AWS_BUCKET_GRPC_STUBS,
+  },
+};
+
+exports.default = async function (orgId = "6ce80f485dae487688c3a083688819bb", serviceId = "test_freecall") {
```
Comment (Vivek205) [inline services/compileGRPCStubs/template/downloadStubFiles.ts]: why this orgId and serviceId is hardcoded. could you check this.

File: utils/LambdaRequestResponse.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { APIGatewayEvent } from "aws-lambda";
+
+export const request = (event: APIGatewayEvent) => {
+  return JSON.parse(event.body) || {};
+};
+
+export const response = (
+  message: string,
```
Comment (Vivek205) [inline utils/LambdaRequestResponse.ts]: add return types to the functions

Comment (rajeevbbqq) [reply utils/LambdaRequestResponse.ts]: Added in the latest commit

File: services/compileGRPCStubs/template/downloadStubFiles.ts (hunk 2 above)
Comment (rajeevbbqq) [reply services/compileGRPCStubs/template/downloadStubFiles.ts]: Hardcoded, for now, the API still gets merged to Ropsten, till then we can only hardcode the values. Parallelly I'll update to dynamic one

File: handlers/boilerplatecode.ts (hunk 1 above)
Comment (rajeevbbqq) [reply handlers/boilerplatecode.ts:4]: Added in the latest commit

**Review by Vivek205 — APPROVED** [review]

**Review by Vivek205 — APPROVED** [review]

**Pull request merged**